easybuild/easyconfigs/p/PyTorch/PyTorch-1.9.0-fosscuda-2020b-imkl.eb

```diff
@@ -0,0 +1,129 @@
+version = '1.9.0'
+versionsuffix = '-imkl'
+
+homepage = 'https://pytorch.org/'
+description = """Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+PyTorch is a deep learning framework that puts Python first."""
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/pytorch',
+        'repo_name': 'pytorch',
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+}]
+patches = [
+    'PyTorch-1.6.0_fix-test-dataloader-fixed-affinity.patch',
+    'PyTorch-1.7.0_avoid-nan-in-test-torch.patch',
+    'PyTorch-1.7.0_disable-dev-shm-test.patch',
+    'PyTorch-1.7.1_correctly-pass-jit_opt_level.patch',
+    'PyTorch-1.7.1_fix-alias-violation-in-bitwise-ops.patch',
+    'PyTorch-1.8.1_dont-use-gpu-ccc-in-test.patch',
+    'PyTorch-1.8.1_fix-arange-on-VSX.patch',
+    'PyTorch-1.8.1_fix-faulty-asserts-and-skip-test.patch',
+    'PyTorch-1.8.1_increase-distributed-test-timeout.patch',
+    'PyTorch-1.8.1_skip_dist_autograd_sync_streams.patch',
+    'PyTorch-1.9.0_avoid-failures-in-test_unary_ufuncs.patch',
+    'PyTorch-1.9.0_fix-min-amount-of-devices-for-test.patch',
+    'PyTorch-1.9.0_fix-testnn-on-A100.patch',
+    'PyTorch-1.9.0_fix-use-after-destruct-in-cudaipctypes.patch',
+    'PyTorch-1.9.0_fix-vsx-vector-functions.patch',
+    'PyTorch-1.9.0_increase-test-cuda-tolerance.patch',
+    'PyTorch-1.9.0_increase-tolerance-for-distributed-tests.patch',
+    'PyTorch-1.9.0_limit-world-size-for-zero-redundancy-opt-test.patch',
+    'PyTorch-1.9.0_skip-lstm-serialization-test.patch',
+    'PyTorch-1.9.0_skip-nccl-error-tests.patch',
+]
+checksums = [
+    None,  # can't add proper SHA256 checksum, because source tarball is created locally after recursive 'git clone'
+    # PyTorch-1.6.0_fix-test-dataloader-fixed-affinity.patch
+    'a4208a46cd2098744daaba96cebb96cd91166f8fc616924315e05974bad80c67',
+    'b899aa94d9e60f11ee75a706563312ccefa9cf432756c470caa8e623991c8f18',  # PyTorch-1.7.0_avoid-nan-in-test-torch.patch
+    '622cb1eaeadc06e13128a862d9946bcc1f1edd3d02b259c56a9aecc4d5406b8a',  # PyTorch-1.7.0_disable-dev-shm-test.patch
+    # PyTorch-1.7.1_correctly-pass-jit_opt_level.patch
+    'd4d967d47f8a6172fcbf57f0a61835482968850967c4fdb01108b720696a988d',
+    # PyTorch-1.7.1_fix-alias-violation-in-bitwise-ops.patch
+    'e92f054f1297df83ace901e7af38222787b709ee29580f5f2b89a300ca03666b',
+    '89ac7a8e9e7df2e64cf8404fe3a279f5e9b759fee41c9de3aaff9c22f385c2c6',  # PyTorch-1.8.1_dont-use-gpu-ccc-in-test.patch
+    'eca718ce0ecc61b57659808179cc660919b3c25b5dce326f51c1cc5a5bfb9829',  # PyTorch-1.8.1_fix-arange-on-VSX.patch
+    # PyTorch-1.8.1_fix-faulty-asserts-and-skip-test.patch
+    '1601eacd336e01176bdbdbc5a2207743dc2999b535a738a8e36b3da3e8a2f3b6',
+    # PyTorch-1.8.1_increase-distributed-test-timeout.patch
+    'c94eda2289692d00873e2ae8cecaaf7a1e5e657dfa4cfcd94b56093f6f4766ad',
```

Comment:
Will conflict with https://github.com/easybuilders/easybuild-easyconfigs/pull/13700
So whichever gets merged first requires a change to the other